### PR TITLE
[WIP] 1.11.2 Port

### PR DESCRIPTION
**Pull request by Mohron**
As discussed in IRC, this PR is to help provide you better oversight of my progress.

### CODE REVIEW

File: src/main/java/crazypants/enderio/BlockEio.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ protected BlockEio(@Nonnull String name, @Nullable Class<T> teClass, @Nonnull Ma
   }
 
   @Override
-  public boolean onBlockActivated(World world, BlockPos pos, IBlockState state, EntityPlayer entityPlayer, EnumHand hand, @Nullable ItemStack heldItem,
+  public boolean onBlockActivated(World world, BlockPos pos, IBlockState state, EntityPlayer entityPlayer, EnumHand hand, @Nonnull ItemStack heldItem,
```
Comment (TomyLobo) [inline src/main/java/crazypants/enderio/BlockEio.java]: Really, the held item can't be null anymore in 1.11?

Comment (Mohron) [inline src/main/java/crazypants/enderio/BlockEio.java]: ItemStack.EMPTY replaced null item stacks

Comment (TomyLobo) [inline src/main/java/crazypants/enderio/BlockEio.java]: I wasn't aware, good to know :)

**Comment by Mohron** [comment]
You must have caught up to my crappy progress, eh... I got caught up in other things unfortunately.

**Comment by HenryLoenwind** [comment]
yeah, I wanted to get started now. If you find some time again, plz contact me in IRC to coordinate. Thanks for the effort!

**Pull request merged**